charts/gardener-dashboard/templates/vpa.yaml

```diff
@@ -0,0 +1,24 @@
+# SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.vpa }}
+apiVersion: autoscaling.k8s.io/v1beta2
+kind: VerticalPodAutoscaler
+metadata:
+  name: gardener-dashboard-vpa
+  namespace: {{ .Release.Namespace }}
+spec:
+  targetRef:
+    apiVersion: {{ include "deploymentversion" . }}
+    kind: Deployment
+    name: gardener-dashboard
+  updatePolicy:
+    updateMode: Auto
+  resourcePolicy:
+    containerPolicies:
+      - containerName: gardener-dashboard
+        minAllowed:
```

Comment:
don't we want to set sane `maxAllowed` values?

Reply:
The Gardener also only specifies `minAllowed` values. Why do you think we should specify `maxAllowed` values?

Reply:
added maxAllowed